sematic/ui/src/hooks/scrollingHooks.ts

```diff
@@ -151,7 +151,7 @@ export function useScrollTracker(
         setHasReachedBottom(
             _hasElementScrolledToBottom(refElement.current)
         );
-    }, [hasReachedBottom, setHasReachedBottom, refElement]);
+    }, [setHasReachedBottom, refElement]);
```

Comment:
Well I see real bugs were caught.

Reply:
This might not cause incorrect behavior, but it is indeed unnecessary.